packages/flutter_tools/lib/src/flutter_manifest/parse_list.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:yaml/yaml.dart';
+
+import 'parse_result.dart';
+
+ParseResult<List<T>> parseList<T>(Object? yamlList, String context, String typeAlias) {
```

Comment:
Is `typeAlias` necessary if you already have the generic `T`?

I'm just noticing that in all of the references, it looks like `T` is the same as `typeAlias`?